include/golioth/client.h

```diff
@@ -79,11 +79,11 @@ struct golioth_psk_credential {
     size_t psk_len;
 };
 
-/// Public Key Infrastructure (PKI) credentials (aka "certificates").
+/// Public Key Infrastructure (PKI) credential (aka "certificate").
 ///
 /// All memory is owned by user and must persist for the lifetime
 /// of the golioth client.
-typedef struct {
+struct golioth_pki_credential {
```

Comment:
Same comment as I left on the PSK credential. It would be to mention the reasoning for the name change (dropping the `s`) in the commit message.